netjsonconfig/backends/airos/interface.py

```diff
@@ -0,0 +1,127 @@
+from ipaddress import ip_interface
+
+
+def autonegotiation(interface):
+    """
+    Return the configuration for ``autoneg`` on interface
+    """
+    if interface.get('autoneg'):
+        return 'enabled'
+    else:
+        return 'disabled'
+
+
+def bridge(interfaces):
+    """
+    Return the bridge interfaces from the interfaces list
+    """
+    return [i for i in interfaces if i['type'] == 'bridge']
+
+
+def bssid(interface):
+    """
+    Return the interface bssid
+    """
+    return interface['wireless'].get('bssid', '')
+
+
+def encryption(interface):
+    """
+    Return the encryption dict for a wireless interface
+    """
+    return interface['wireless']['encryption']
```

Comment:
the encryption key is not mandatory, see: http://netjson.org/rfc.html#rfc.section.5.4.2
if encryption is omitted it means encryption off. I discovered this because I got an exception while testing.

Please, before fixing this issue could you first create a failing test?